components/pages/articlePageLayout/ArticlePageLayout.tsx

```diff
@@ -0,0 +1,67 @@
+import React from "react"
+
+import ParagraphResolver from "@/components/paragraphs/ParagraphResolver"
+import ImageBaseWithPlaceholder from "@/components/shared/image/ImageBaseWithPlaceholder"
+import { Maybe, MediaImage, NodeGoArticle } from "@/lib/graphql/generated/dpl-cms/graphql"
+
+export type TArticlePageLayoutProps = {
+  goArticleImage?: Maybe<MediaImage>
+} & NodeGoArticle
+
+function ArticlePageLayout({ pageData }: { pageData: TArticlePageLayoutProps }) {
+  const { paragraphs } = pageData
+  const { goArticleImage } = pageData
+
+  if (!paragraphs) {
+    return null
+  }
+
+  const mediaImage = goArticleImage?.mediaImage
+
+  const localeDateString = new Date(
+    pageData.publicationDate.timestamp as string
+  ).toLocaleDateString("da-DK", {
+    day: "numeric",
+    month: "long",
+    year: "numeric",
+  })
```

Comment:
I think we should:
* Make sure that the date is a Date object in the `loadArticle` function
* Introduce a date formatting helper in `/lib` with date formats we can use throughout the application
*  Use a formatting function from the helper here